libnmstate/state.py

```diff
@@ -771,6 +772,20 @@ def merge_route_rules(self, other_state):
                 )
             }
 
+    def _normalize_linux_bridge_port_vlan(self):
+        linux_bridges = [
+            iface
+            for iface in self.interfaces.values()
+            if iface[Interface.TYPE] == InterfaceType.LINUX_BRIDGE
+        ]
```

Comment:
This fits a generator. It can be converted to one by just replacing the square brackets with regular ones (`()`).